packages/pn-commons/src/components/CustomDatePicker.tsx

```diff
@@ -19,15 +16,17 @@ const locales: { [key: string]: Locale } = {
 
 export type DatePickerTypes = Date | null;
 const CustomDatePicker = (
-  props: DesktopDatePickerProps<DatePickerTypes> &
+  props: DatePickerProps<DatePickerTypes> &
     React.RefAttributes<HTMLDivElement> & { language?: string }
 ) => {
   const language = props.language ? props.language : 'it';
   return (
     <LocalizationProvider dateAdapter={AdapterDateFns} adapterLocale={locales[`${language}`]}>
-      <DesktopDatePicker
+      <DatePicker
```

Comment:
The view on mobile devices is not good for our needs. Datepicker popup has a toolbar on top and has actions on bottom. You can remove it with slotProps https://mui.com/x/react-date-pickers/custom-layout/.

TIPS: define property slotProps in witch you remove toolbar and action bar. pay attention of the slotProps defined from the outside. you have to merge the slotProps defined here whit those from the outside. To hide the action bar you must pass empty actions array to the slotProps.

On the mobile view, the datepicker doesn't close on selection. Check on https://mui.com/x/api/date-pickers/date-picker/ how to do that.

Reply:
done